src/types/widgets.ts

```diff
@@ -76,6 +76,7 @@ export interface ExpandedTileOptions {
   show_votes: boolean
   show_cross_sellers: boolean
   show_add_to_cart: boolean
+  auto_play_video: boolean
```

Comment:
We dont need this here (I think) as the autoplay will only run inline.
Autoplay is enabled by default for all widget expanded tiles anyway.